### ISSUE TITLE
Use only real version names, not `next`, for doc version subdirectories under /content

### ISSUE THREAD

**Issue by chalin** [etcd-io/website#252]
@spzala et al: I'd like to propose that all our doc version subfolders name the version they are targeting. That is currently the case except for the special `/docs/next`. Some drawbacks of using `/docs/next` as part of permalinks include the following:

- This muddles analytics
- It makes releasing a new version of the docs a bit more work (for details, see the following thread: https://github.com/etcd-io/website/pull/203#discussion_r618467132)
- In my opinion, it can confuse site users as explained next.

Let's suppose that we just released v3.5 docs, under the current scheme we'd already have a `/docs/next` even though there are no v3.6.x-DRAFT docs. Under the new scheme we can either, not create v3.6 until we have v3.6 docs, or we can immediately create the branch but mark it as draft and not publish it (until we have actual v3.6 specific changes).

We can, of course, keep `/docs/next` as a virtual path that is redirected to the "next" release of the docs.

Thoughts?

Related: #220, though that concerns the displayed name, not permalink segment names.

/cc @nate-double-u

**Comment by nate-double-u** [comment]
Setting e1-hours and p1-high. I don't think this will take that long to do, but it's important that we make a decision around how we name our upcoming versions moving forward.

**Comment by nate-double-u** [comment]
I agree with the proposal that all our doc version subfolders **name the version** they are targeting.

**Comment by chalin** [comment]
Think about this more, an alternative would be to keep `next` but define a redirect rule based on the eventual version that `next` will be.

**Comment by chalin** [comment]
All: as a step in this direction, we'll be implementing the proposal (in the opening comment) as part of the preparation for the v3.5 docs; see #332. /cc @spzala @ptabor

**Comment by nate-double-u** [comment]
Note, currently there is no `next`. Since the v3.5 release, `next` points to the same folder as `latest`.

**Comment by nate-double-u** [comment]
Related conversation:
https://github.com/etcd-io/etcd/pull/13110#issuecomment-862726764 (@ptabor)
